src/Stratis.Bitcoin/P2P/Peer/NetworkPeerServer.cs

```diff
@@ -12,9 +12,6 @@
 
 namespace Stratis.Bitcoin.P2P.Peer
 {
-    public delegate void NetworkPeerServerNodeEventHandler(NetworkPeerServer sender, NetworkPeer peer);
-    public delegate void NetworkPeerServerMessageEventHandler(NetworkPeerServer sender, IncomingMessage message);
-
     public class NetworkPeerServer : IDisposable
```

Comment:
I suggest we rename this to `PeerListener` this will be consistent with what @fassadlr has the `PeerConnector`
We then need to think if we rename `PeerConnector` to `NetworkPeerConnector`

Reply:
sadly NetworkPeerListener already exists, using either NetworkPeerListener or PeerListener for this would be highly confusing at this moment

so maybe at later phase of refactoring?

Reply:
ah ok yes good point.